setup.py

```diff
@@ -14,6 +14,7 @@
 
 install_requires = [
     'draftjs_exporter==0.7.0',
+    'jinja2>=2.9.5,<3'  # TODO: remove when it's not necessary (https://github.com/springload/wagtaildraftail/issues/13)
```

Comment:
Well, as far as we use WagtaildraftTextBlock there is no way to avoid having Jinja2 as a dependency. However, it appears we don't really need Jinja2 since we only use MarkSafe.Markup  class. Furthermore, we probably don't even need MarkSafe as a dependency, because we apparently don't use its features - I'll try to go with django.utils.safestring.mark_safe instead